packages/patternfly-3/patternfly-react/src/components/DualListSelector/helpers.js

```diff
@@ -38,22 +38,34 @@ export const makeAllItemsVisible = list =>
 export const sortItems = (items, sortFactor = 'label') =>
   items.sort((a, b) => (a[sortFactor].toLowerCase() > b[sortFactor].toLowerCase() ? 1 : -1));
 
-export const arrangeArray = ({ items, sortBy, isSortAsc = true, isMainChecked = false }) => {
+export const shouldItemBeChecked = ({ item, resetAllSelected, isMainChecked }) => {
```

Comment:
why take a object with three fields rather than 3 parameters?

Reply:
oops, copy-pasted it from another function signature with a lot of properties,
I don't mind changing it, 
btw from what number of arguments do we usually provide objects instead?

Reply:
I'm not sure we ever should in helper functions

Reply:
no problem, opening a PR to refactor that.

Reply:
Added the [refactor PR](https://github.com/patternfly/patternfly-react/pull/1190) now